cmd/createcluster.go

```diff
@@ -119,6 +121,7 @@ func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.StringVar(&config.testnetConfig.GenesisForkVersionHex, "testnet-fork-version", "", "Genesis fork version of the custom test network (in hex).")
 	flags.Uint64Var(&config.testnetConfig.ChainID, "testnet-chain-id", 0, "Chain ID of the custom test network.")
 	flags.Int64Var(&config.testnetConfig.GenesisTimestamp, "testnet-genesis-timestamp", 0, "Genesis timestamp of the custom test network.")
+	flags.IntSliceVar(&config.DepositAmounts, "deposit-amounts", nil, "List of partial deposit amounts in gwei. Values must sum up to exactly 32ETH.")
```

Comment:
can we have the values of deposit amounts in ether rather than gwei? that maybe more user friendly. For example, `--deposit-amounts=2,8,22` versus `--deposit-amounts=2000000000,8000000000,22000000000`

Reply:
This was suggested by Oisin, see the design doc.

Reply:
Thanks, added a comment in design doc.